manifests/repository.pp

```diff
@@ -11,7 +11,7 @@
   $url = "http://download.bareos.org/bareos/release/${release}/"
 
   $os = $::operatingsystem
```

Comment:
this should fix the error and adds indirect support for old and new puppet release $os = pick($facts['os']['name'], $::operatingsystem)

Reply:
works with centos 7

Reply:
this is fine, but breaks checks and compatibility with puppet 3.x